<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>loginAndSignUpDialog</class>
 <widget class="QDialog" name="loginAndSignUpDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>301</width>
    <height>602</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>10000000</width>
    <height>10000000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>loginAndSignUpDialog</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>241</width>
     <height>501</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="loginPage">
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>227</width>
       <height>141</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="emailVLayoutLogIn">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item>
         <widget class="QLabel" name="emailLabelLogIn">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>81</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>81</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>Email</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="emailLineEditLogIn">
          <property name="minimumSize">
           <size>
            <width>221</width>
            <height>31</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>221</width>
            <height>31</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="passwordVLayoutLogIn">
        <item>
         <widget class="QLabel" name="passwordLabelLogIn">
          <property name="minimumSize">
           <size>
            <width>63</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>63</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>Password</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="passwordLineEditLogIn">
          <property name="minimumSize">
           <size>
            <width>221</width>
            <height>31</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>211</width>
            <height>31</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="signUpPage">
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>225</width>
       <height>461</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetFixedSize</enum>
      </property>
      <item>
       <layout class="QVBoxLayout" name="firstNameVLayout">
        <property name="sizeConstraint">
         <enum>QLayout::SetMinimumSize</enum>
        </property>
        <item>
         <widget class="QLabel" name="firstNameLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>81</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>81</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>First Name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="firstNameLineEdit">
          <property name="minimumSize">
           <size>
            <width>221</width>
            <height>31</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>221</width>
            <height>31</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="lastNameVLayout">
        <item>
         <widget class="QLabel" name="lastNameLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>81</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>81</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>Last Name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lastNameLineEdit">
          <property name="minimumSize">
           <size>
            <width>221</width>
            <height>31</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>221</width>
            <height>31</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="emailVlayout">
        <item>
         <widget class="QLabel" name="emailLabelSignUp">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>81</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>81</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>Email</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="emailTextEditSignUp">
          <property name="minimumSize">
           <size>
            <width>221</width>
            <height>31</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>221</width>
            <height>31</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="passwordVLayoutSignUp">
        <item>
         <widget class="QLabel" name="passwordLabelSignUp">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>63</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>63</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>Password</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="passwordLineEditSignUp">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>221</width>
            <height>31</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="confirmPasswordVLayout">
        <item>
         <widget class="QLabel" name="confirmPasswordLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>130</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>130</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>Confirm Password</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="confirmPasswordLineEdit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>221</width>
            <height>31</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="comboBoxVLayout">
        <item>
         <widget class="QLabel" name="comboBoxLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Role</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <item>
           <property name="text">
            <string>Lawyer</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Supervisor</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="supervisorVLayout">
        <item>
         <widget class="QLabel" name="supervisorLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>Supervisor</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="supervisorLineEdit">
          <property name="minimumSize">
           <size>
            <width>221</width>
            <height>31</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>221</width>
            <height>31</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>520</y>
     <width>196</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="loginAndSignUpButtonsLayout">
    <property name="spacing">
     <number>7</number>
    </property>
    <item>
     <widget class="QPushButton" name="loginButton">
      <property name="text">
       <string>Sign In</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="loginAndSignUpHSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="signUpButton">
      <property name="text">
       <string>Sign Up</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
